pkg/controller/acm/controller.go

```diff
@@ -57,9 +57,12 @@ const (
 	errRemoveTagsFailed = "failed to remove tags for Certificate"
 )
 
+// ControllerName of this controller.
+var ControllerName = managed.ControllerName(v1beta1.CertificateGroupKind)
+
 // SetupCertificate adds a controller that reconciles Certificates.
 func SetupCertificate(mgr ctrl.Manager, o controller.Options) error {
-	name := managed.ControllerName(v1beta1.CertificateGroupKind)
```

Comment:
for what reason we need this change ?

Reply:
Since I already Add `ControllerName`, so I think it is better to use that added var to avoid duplication and need to change both of these two places later if we need to change the name.

Reply:
I would prefer to remove this ,and leave it as is

Reply:
reverted

Reply:
can you check again ? you have this kind of configuration in each and every resource - please cleanup =)

Reply:
I have checked it.
Note in `pkg/controller/aws.go` there are two duplicate setups before:
first:
https://github.com/crossplane-contrib/provider-aws/blob/ca0154a118ac2a0f377a56f8ffe9f4c447a9b20c/pkg/controller/aws.go#L239-L240
and:
https://github.com/crossplane-contrib/provider-aws/blob/ca0154a118ac2a0f377a56f8ffe9f4c447a9b20c/pkg/controller/aws.go#L332-L333

I have removed this duplicate and made sure all setup remains. 

> please cleanup =)

Do you mean to rebase the commits?